src/service/action-impl.js

```diff
@@ -278,34 +253,31 @@ export class ActionService {
 
   /**
    * Given a browser 'change' or 'input' event, add `details` property
-   * containing the relevant information for the change that generated
-   * the initial event.
+   * containing the relevant information for the change that generated it.
    * @param {!ActionEventDef} event
    */
   addInputDetails_(event) {
     const detail = map();
```

Comment:
Good idea! Cast instead of using `dict()` since we want a prototype-less object.